julia/deps/build.jl

```diff
@@ -26,14 +26,35 @@ libmxnet_detected = false
 libmxnet_curr_ver = get(ENV, "MXNET_COMMIT", "master")
 curr_win = "20190608"  # v1.5.0
 
+# TODO: remove MXNET_HOME backward compatibility in v1.7
 if haskey(ENV, "MXNET_HOME")
-  MXNET_HOME = ENV["MXNET_HOME"]
-  @info("MXNET_HOME environment detected: $MXNET_HOME")
+  @warn "The environment variable `MXNET_HOME` has been renamed, please use `MXNET_ROOT` instead."
+end
+
+# TODO: remove MXNET_HOME backward compatibility in v1.7
```

Comment:
2.0

Can you add it to the 2.0 tracking issue?

Reply:
ah, sure (https://github.com/apache/incubator-mxnet/issues/9686#issuecomment-529131708)
```suggestion
# TODO: remove MXNET_HOME backward compatibility in v2.0
```